test/external-modules/die-with-dignity/src/javaRestTest/java/org/elasticsearch/qa/die_with_dignity/DieWithDignityIT.java

```diff
@@ -30,6 +27,24 @@
 public class DieWithDignityIT extends ESRestTestCase {
 
     public void testDieWithDignity() throws Exception {
+        // there should be an Elasticsearch process running with the die.with.dignity.test system property
+        assertBusy(() -> {
```

Comment:
Why is this stall necessary? Doesn't the rest test infrastructure not run the tests until ES is up?

Reply:
The simple explanation is that I copy/pasted the code below that checks that the die-with-dignity instance has been killed, with the intention of modifying it to remove the busy assert, and change it to a check that the die-with-dignity instance is up. Except, I forgot to remove the busy assert. 🤦‍♀️

I've removed this now in 3dd8db62638691bcb96f2813bf719c82b4546655.